includes/class-wc-calypso-bridge-woocommerce-admin-features.php

```diff
@@ -50,6 +50,10 @@ public function filter_wc_admin_enabled_features( $features ) {
 			$features[] = 'remote-inbox-notifications';
 		}
 
+		if ( ! array_key_exists( 'navigation', $features ) && 'yes' === get_option( 'woocommerce_navigation_enabled', 'yes' ) ) {
```

Comment:
This logic seems to be off. Although the option `woocommerce_navigation_enabled` will be on by default (which we need to do), it appears the feature will be turned off when the option is set to "no".

We should
* Initialize the option to "yes" if it doesn't exist, but respect it if its "no" or already set to "yes"
* Force the feature flag on regardless of the state of the option

Reply:
ok, it further appears that I have confused myself! This is working well and I don't think requires a change